Common/Data/Market/Dividend.cs

```diff
@@ -23,6 +23,25 @@ namespace QuantConnect.Data.Market
     /// </summary>
     public class Dividend : BaseData
     {
+        /// <summary>
+        /// Gets the dividend payment
+        /// </summary>
+        public decimal Distribution
+        {
+            get { return Math.Round(Value, 2); }
```

Comment:
Would rounding in the `set` cover the `get` case?

Reply:
I debated this myself, the issue is that it *can* be set directly via the `Value` property which isn't virtual... in reality, this should be updated to hold it's own storage and not use the `Value` property as it can be confusing